Cargo.toml

```diff
@@ -16,15 +16,20 @@ cosmwasm-std    = "^1.2"
 cw2             = "^1.1"
 cw20            = "^1.1"
 cw721           = { version = "*", path = "./packages/cw721" }
+cw721-016       = { git = "https://github.com/CosmWasm/cw-nfts", tag = "v0.16.0", package = "cw721" } # needed for backwards compatibility and legacy migration
 cw721-base      = { version = "*", path = "./contracts/cw721-base" }
-cw721-base-016  = { version = "0.16.0", package = "cw721-base" }
+cw721-base-015  = { git = "https://github.com/CosmWasm/cw-nfts", tag = "v0.15.0", package = "cw721-base" } # needed for testing legacy migration
+cw721-base-016  = { git = "https://github.com/CosmWasm/cw-nfts", tag = "v0.16.0", package = "cw721-base" } # needed for testing legacy migration
+cw721-base-017  = { git = "https://github.com/CosmWasm/cw-nfts", tag = "v0.17.0", package = "cw721-base" } # needed for testing legacy migration
+cw721-base-018  = { git = "https://github.com/CosmWasm/cw-nfts", tag = "v0.18.0", package = "cw721-base" } # needed for testing legacy migration
 cw-multi-test   = "^0.20"
-cw-ownable      = "^0.5"
+cw-ownable      = { git = "https://github.com/public-awesome/cw-plus-plus.git", branch = "main"} # TODO: switch to official https://github.com/larry0x/cw-plus-plus once merged
```

Comment:
Yes, please switch to the official. Curious why you're not using the latest `cw-ownable` published on crates.io?

Reply:
I did a PR on `cw-ownable` here: https://github.com/public-awesome/cw-plus-plus/pull/1
Basically I had to adjust it, since it is a singleton, but I need it for `creator` and `minter`.
Larry's going to approve this.

Reply:
fyi @larry0x